src/components/CourseMap/index.jsx

```diff
@@ -17,13 +17,18 @@ function CourseMap({ markerData, reload, mapName }) {
     else if (tag === markerData.length - 1) imgURL = endMarker;
     else imgURL = pointMarker;
 
+    let label =
```

Comment:
```suggestion
    const label =
```
이 변수는 `const`로 선언하는 게 좋을 것 같은데, 큰 이슈는 아니니까 일단 머지할게~